### ISSUE TITLE
Proposals for Settings/Profile UI

### ISSUE THREAD

**Issue by dakingha69** [trustlines-network/mobileapp#43]
### Problem
We need new proposals for the UI of the Settings and Profile screen. Especially if and how to merge those screens.

### Suggestion
tbd

**Comment by Kispus** [comment]
Current implementation in app:

![10dd59c6-51d6-4e63-b549-3912c8ea29ac](https://user-images.githubusercontent.com/18598519/33071308-53a6bc26-cebb-11e7-8c6c-c429c40eac57.jpg)
![899bfc2d-2af9-48cd-889a-4a56fa6941d3](https://user-images.githubusercontent.com/18598519/33071309-53d91bd0-cebb-11e7-83c3-0d96e49be573.jpg)
![366127fd-bcde-4d1d-a3d5-1f6fac85c6eb](https://user-images.githubusercontent.com/18598519/33071310-5400e9f8-cebb-11e7-8b3f-50003e327753.jpg)

Current mock-up by Daking:

![screen shot 2017-11-21 at 12 56 52 pm](https://user-images.githubusercontent.com/18598519/33071346-78c7a5d8-cebb-11e7-95b1-69cda5b64dc7.png)

**Comment by Kispus** [comment]
Q @dakingha69: Do we really need individual buttons for 1) Export Private Key 2) Backup Mnemominc 3) Backup Wallet, are these not overlapping?

Missing actions according to current implementation:

-Detailed overview of currency network balances
-Log-out / Sign-out
-Help/Feedback

**Comment by dakingha69** [comment]
@Kispus they are a bit overlapping... I changed it to just *Backup Account* in the mockups. This option should just show the 12 word seed for now.

I also decided to divide between the Profile view and Settings view because it is already implemented like this and the overview of currencey networks balances also need its place.

**New Proposal** (dont mind the font)
![profile](https://user-images.githubusercontent.com/17645687/33122534-c7d3404e-cf77-11e7-9a8f-f972fc358149.png)
![screenshot from 2017-11-22 11-25-46](https://user-images.githubusercontent.com/17645687/33122649-165cc69a-cf78-11e7-8486-0865f68ddea4.png)
![screenshot from 2017-11-22 11-27-41](https://user-images.githubusercontent.com/17645687/33122708-406aabfa-cf78-11e7-9960-b241a465a83a.png)
